eng/Versions.props

```diff
@@ -3,10 +3,10 @@
   <PropertyGroup>
     <MajorVersion>3</MajorVersion>
     <MinorVersion>1</MinorVersion>
-    <PatchVersion>1</PatchVersion>
+    <PatchVersion>2</PatchVersion>
     <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
     <!-- Enable to remove prerelease label. -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
+    <StabilizePackageVersion>true</StabilizePackageVersion>
```

Comment:
Same issue as https://github.com/dotnet/core-setup/pull/8947#discussion_r366540608

Reply:
I will fix it